ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/NodeCve/AffectedNodesTable.cy.jsx

```diff
@@ -0,0 +1,284 @@
+                { rowIndex: 4, expectedFixability: 'Fixable' },
+            ].forEach(({ rowIndex, expectedFixability }) => {
+                cy.findAllByRole('row')
+                    .eq(rowIndex)
+                    .within(() => {
+                        cy.get(`td[data-label="CVE status"]:contains("${expectedFixability}")`);
+                    });
+            });
+        });
+
+        it('should render the highest CVSS score of all matching component vulnerabilities', () => {
+            let id = 0;
+            function createNodeWithComponentCvss(cvssGroups) {
+                id += 1;
+                return mockNode({
+                    id,
+                    nodeComponents: cvssGroups.map((cvssGroup) =>
+                        mockNodeComponent({
+                            nodeVulnerabilities: cvssGroup.map(mockNodeVulnerability),
+                        })
+                    ),
+                });
+            }
+
+            const nodes = [
+                createNodeWithComponentCvss([
+                    [{ cvss: 2.0, scoreVersion: 'V3' }],
+                    [
+                        { cvss: 1.0, scoreVersion: 'V3' },
+                        { cvss: 1.0, scoreVersion: 'V2' },
+                    ],
+                ]),
+                createNodeWithComponentCvss([
+                    [
+                        { cvss: 3.0, scoreVersion: 'V3' },
+                        { cvss: 4.0, scoreVersion: 'V2' },
+                    ],
+                ]),
+                createNodeWithComponentCvss([
+                    [
+                        { cvss: 5.0, scoreVersion: 'V2' },
+                        { cvss: 6.0, scoreVersion: 'V3' },
+                        { cvss: 7.0, scoreVersion: 'V2' },
+                    ],
+                    [
+                        { cvss: 6.0, scoreVersion: 'V3' },
+                        { cvss: 2.0, scoreVersion: 'V2' },
+                    ],
+                ]),
+            ];
+
+            setup({ type: 'COMPLETE', data: nodes });
+
+            // Avoid the 0th row which is the header, and then check that the highest CVSS score is rendered correctly
+            // for each row
+            [
+                { rowIndex: 1, expectedCVSS: '2.0', expectedVersion: 'V3' },
+                { rowIndex: 2, expectedCVSS: '4.0', expectedVersion: 'V2' },
+                { rowIndex: 3, expectedCVSS: '7.0', expectedVersion: 'V2' },
+            ].forEach(({ rowIndex, expectedCVSS, expectedVersion }) => {
+                cy.findAllByRole('row')
+                    .eq(rowIndex)
+                    .within(() => {
+                        cy.get(
+                            `td[data-label="CVSS score"]:contains("${expectedCVSS} (${expectedVersion})")`
+                        );
+                    });
+            });
+        });
+
+        it('should allow expanding and collapsing the nested component table', () => {
+            // TODO: Implement this test
```

Comment:
Suggestion: add a datestamp to TODO out of habit, to help identify stale TODOs later (probably wouldn't be a problem here)

Reply:
Good idea. I'll have to make a ticket to address all the floating TODOs (either fix or add a date to) in Node/Platform CVEs before closing out the epic.